web-components/src/theme/muiTheme.ts

```diff
@@ -63,78 +60,77 @@ const regenTheme = createTheme({
   },
   spacing: (factor: number) => `${0.25 * factor}rem`,
   typography: {
-    fontFamily: ['"Lato"', '-apple-system', 'sans-serif'].join(','),
+    fontFamily: defaultFontFamily,
     h1: {
-      [defaultTheme.breakpoints.up('sm')]: {
-        fontSize: '3rem',
-      },
-      [defaultTheme.breakpoints.down('sm')]: {
-        fontSize: '2rem',
-      },
-      fontFamily: headerFontFamily,
+      ...headerDefaults,
+      fontSize: pxToRem(48),
+      lineHeight: '130%',
     },
     h2: {
-      [defaultTheme.breakpoints.up('sm')]: {
-        fontSize: '2.375rem',
-      },
-      [defaultTheme.breakpoints.down('sm')]: {
-        fontSize: '1.5rem',
-      },
+      ...headerDefaults,
+      fontSize: pxToRem(38),
+      lineHeight: '130%',
     },
     h3: {
-      [defaultTheme.breakpoints.up('sm')]: {
-        fontSize: '2rem',
-      },
-      [defaultTheme.breakpoints.down('sm')]: {
-        fontSize: '1.5rem', // possible override: 1.3125rem
-      },
+      ...headerDefaults,
+      fontSize: pxToRem(32),
+      lineHeight: '140%',
     },
     h4: {
-      [defaultTheme.breakpoints.up('sm')]: {
-        fontSize: '1.5rem',
-      },
-      [defaultTheme.breakpoints.down('sm')]: {
-        fontSize: '1.3125rem', // possible override: 1.125rem
-      },
+      ...headerDefaults,
+      fontSize: pxToRem(24),
+      lineHeight: '145%',
     },
     h5: {
-      [defaultTheme.breakpoints.up('sm')]: {
-        fontSize: '1.3125rem',
-      },
-      [defaultTheme.breakpoints.down('sm')]: {
-        fontSize: '1.125rem',
-      },
+      ...headerDefaults,
+      fontSize: pxToRem(21),
+      lineHeight: '150%',
     },
     h6: {
-      fontSize: '1.125rem',
+      ...headerDefaults,
+      fontSize: pxToRem(18),
+      lineHeight: '150%',
     },
-    subtitle1: {
-      [defaultTheme.breakpoints.up('sm')]: {
-        fontSize: '2.375rem',
-      },
-      [defaultTheme.breakpoints.down('sm')]: {
-        fontSize: '2rem',
-      },
+    textXLarge: {
+      fontSize: pxToRem(22),
     },
-    subtitle2: {
-      [defaultTheme.breakpoints.up('sm')]: {
-        fontSize: '1.125rem',
-      },
-      [defaultTheme.breakpoints.down('sm')]: {
-        fontSize: '0.875rem',
-      },
+    textLarge: {
+      fontSize: pxToRem(18),
     },
+    // MUI defaults to 'body1' for Typography text and I can't figure out how to
+    // override - this represents the equivalent of `textMedium`
     body1: {
```

Comment:
See the comment above. It's a bit odd here in the theme, however we shouldn't ever really use these variant values directly.

I initially set _all_ of the font sizes as variants, which had the advantage of being able to do things like `<Link variant="subtitleLarge" />` (a link formatted as a large subtitle), but it got more complex with edge cases and didn't keep parity with the Figma values